src/test/java/hw1/dataProviders/AdditionDP.java

```diff
@@ -0,0 +1,22 @@
+package hw1.dataProviders;
+
```

Comment:
**Questions:**
1) Why you decide to place calculator-1.0.jar in main/resources but not in test/resources?
2) What is "etc" in gitignore (line 4)? Remove or clarify pls :)
3) Why you test only `int`? As I know calculator also can operate with `long` and `double`

Reply:
Answers:
1)The requirements said that it was necessary to add-Library calculator-1.0.jar should be added to the project as maven dependence. I didn't know how best to make this point. It turns out that I should add the file to test / resources?
2)I completely agree
3)It was convenient for me to work this way, because the commas after the number are rather intensely straining my eyes. But I can add

Reply:
1)I thought that the code was packed in the archive, but it is not a test, this code describes the logic of the program. On the other hand, if we consider it as a Tet artifact, then we can put it in the test. Who adheres to what logic. Depends on the situation. I would leave him in  main / resources